src/models/flight_control/FGLinearActuator.h

```diff
@@ -0,0 +1,246 @@
+     *      <module>10</module>
+     *      <rate>0.2</rate>
+     *      <lag>8.0</lag>
+     *  </linear_actuator>
+     * 
+     * @endcode
+     * 
+     * The gyrocompass from a rotation with a value from 0 to 259 degrees.
+     * When it returns to zero, if it is made more realistic by means of an actuator,
+     * there is a jump of the disk which performs a complete rotation of 360 °.
+     * By activating a linear actuator followed by an actuator
+     * it is possible to obtain a very realistic result.
+     * 
+     * @code
+     * 
+     * <linear_actuator name="gyrocompass-magnetic-deg-linear">
+     *      <input>gyrocompass-magnetic-deg</input>
+     *      <module>360</module>
+     *  </linear_actuator>
+     * 
+     *  <actuator name="gyrocompass-magnetic-deg-linear-actuator">
+     *      <input>gyrocompass-magnetic-deg-linear</input>
+     *      <lag>2.0</lag>
+     *      <rate_limit>100</rate_limit>
+     *      <bias>0.1</bias>
+     *      <deadband_width>1</deadband_width>
+     *      <hysteresis_width>0.5</hysteresis_width>
+     *  </actuator>
+     * 
+     * @endcode
+     * 
+     * Count steps with memory:
+     * If you use a button or switch to advance a mechanism, we can build a step counter.
+     * In this case the module is 1 and the rate 1. The verse is positive (increasing).
+     * A pulse counter (for example the count of the switched-on states of a switch with values 1 and 0),
+     * the module must be 1 in that each step must advance its value by one unit.
+     * The verse is "1" because it has to accept only increasing values (as in a clock escapement).
+     * The gain is 0.5 because, in similitude to an escapement of a clock,
+     * the gear makes two steps for a complete rotation of the pendulum.
+     *
+     * @code
+     * 
+     *  <linear_actuator name="switch-increase-summer">
+     *     <input>systems/gauges/PHI/doppler/switch-increase</input>
+     *      <module>1</module>
+     *      <rate>1</rate>
+     *      <versus>1</versus>
+     *      <gain>0.5</gain>
+     *  </linear_actuator>
+     *  
+     * @endcode
+     * 
+     * 
+     * <pre>
+     *    Notes:
+     * </pre>
+     * 
+     *    @author Adriano Bassignana
+     */
+    
+    /*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+     * CLASS DECLARATION
+     * %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+    
+    class FGLinear_Actuator : public FGFCSComponent
+    {
+    public:
+        /** Constructor.
+         *      @param fcs a pointer to the parent FGFCS object.
+         *      @param element a pointer to the configuration file node. */
+        FGLinear_Actuator(FGFCS* fcs, Element* element);
+        /// Destructor
+        ~FGLinear_Actuator();
+        
+        /// The execution method for this FCS component.
+        bool Run(void);
+        
+    private:
+        
+        simgear::PropertyObject<double> setProperty;
```

Comment:
Use `FGPropertyNode_ptr` instead
```suggestion
        FGPropertyNode_ptr setProperty;
```